.github/project.yml

```diff
@@ -1,5 +1,5 @@
 name: MockServer Quarkus test extension
 release:
-  current-version: "0"
-  next-version: "999-SNAPSHOT"
+  current-version: 0.1.0
+  next-version: 0.2.0-SNAPSHOT
```

Comment:
```suggestion
  current-version: "0.1.0"
  next-version: "0.2.0-SNAPSHOT"
```

Reply:
I should first read the info before I merge it. I will create 0.2.0 with double quotes